be/src/udf/java/java_native_method.cpp

```diff
@@ -0,0 +1,70 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021 StarRocks Limited.
+
+#include "udf/java/java_native_method.h"
+
+#include "column/column.h"
+#include "column/column_helper.h"
+#include "column/column_visitor_adapter.h"
+#include "column/nullable_column.h"
+#include "column/vectorized_fwd.h"
+#include "gutil/casts.h"
+#include "runtime/primitive_type.h"
+
+namespace starrocks::vectorized {
+
+class getColumnAddrVistor : public ColumnVisitorAdapter<getColumnAddrVistor> {
+public:
+    getColumnAddrVistor(jlong* jarr) : ColumnVisitorAdapter(this), _jarr(jarr) {}
+
+    Status do_visit(const NullableColumn& column) {
+        const auto& null_data = column.immutable_null_column_data();
+        _jarr[_idx++] = reinterpret_cast<int64_t>(null_data.data());
+        return column.data_column()->accept(this);
+    }
+
+    Status do_visit(const BinaryColumn& column) {
```

Comment:
Support `BinaryColumnBase`, which including `BinaryColumn` and `LargeBinaryColumn`?

Reply:
Now we we only support BinaryColumn in Expression.